fuzzing/README.md

```diff
@@ -5,21 +5,24 @@ This repository contains several fuzzing targets that can be used with generic
 fuzzing engines like AFL and libFuzzer.
 
 These targets are built by passing the --enable-fuzz-targets option to the
```

Comment:
could use backticks around `--enable..`, same for `make fuzz_targets` below etc.

Reply:
Indeed, thanks! Done that now!